node/src/actors/sessions_manager/mod.rs

```diff
@@ -50,6 +50,8 @@ pub struct SessionsManager {
     logging_messages: HashSet<String>,
     // Reference to config
     config: Option<Arc<Config>>,
+    // SuperBlock target
+    superblock_beacon_target: Option<CheckpointBeacon>,
```

Comment:
I wonder why the members of this `enum` are not documented with `///` :thinking: